### PR TITLE
Port storage addons to charts repo from kubeaddons-config

### PR DESCRIPTION
**Pull request by gpaul**
This PR ports storage addons from manifest-based addons defined in kubeaddons-config to helm charts defined in the charts repository.

### CODE REVIEW

**Comment by gpaul** [comment]
What does the "blocked" label mean in this case?

File: stable/awsebscsiprovisioner/templates/storageclass.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: awsebscsiprovisioner
+  annotations:
+    kubernetes.io/description: AWS EBS CSI provisioner StorageClass
+    storageclass.kubernetes.io/is-default-class: {{ .Values.storageclassdefault | default "true" | quote }}
+provisioner: ebs.csi.aws.com
+volumeBindingMode: WaitForFirstConsumer
```
Comment (hectorj2f) [inline stable/awsebscsiprovisioner/templates/storageclass.yaml]: Please, add this property to be configurable and also `reclaimPolicy: Delete` too.

**Comment by gpaul** [comment]
If we're going to do more iteration on this PR and it's associated ones in our other repositories, we could consider taking on board the changes suggested by @alejandroEsc in https://github.com/mesosphere/charts/pull/14

File: stable/localvolumeprovisioner/values.yaml [hunk 2]
```diff
@@ -1 +1,4 @@
-storageclassdefault: true
+storageclass:
```
Comment (nfnt) [inline stable/localvolumeprovisioner/values.yaml]: This change isn't compatible with what is checked for in https://github.com/mesosphere/kubeaddons/blob/master/pkg/utils/supported_addons.go#L348

**Pull request merged**